extensions/cricketcast/src/index.jsx

```diff
@@ -0,0 +1,74 @@
+import { ActionPanel, OpenInBrowserAction, List, showToast, ToastStyle, CopyToClipboardAction } from "@raycast/api";
+import { useEffect, useState } from "react";
+import convert from "xml-js";
+import fetch from "node-fetch";
+
+function Actions({ item }) {
+  return (
+    <ActionPanel title={item.title}>
+      <ActionPanel.Section>{item.link && <OpenInBrowserAction url={item.link} />}</ActionPanel.Section>
+      <ActionPanel.Section>
+        {item.link && (
+          <CopyToClipboardAction content={item.link} title="Copy Link" shortcut={{ modifiers: ["cmd"], key: "." }} />
+        )}
+      </ActionPanel.Section>
+    </ActionPanel>
+  );
+}
+
+export default function Command() {
+  const [state, setState] = useState({});
+
+  useEffect(() => {
+    async function fetchStories() {
+      try {
+        let XMLdata = await fetch("http://static.cricinfo.com/rss/livescores.xml").then((r) => r.text());
+        let JSONdata = JSON.parse(await convert.xml2json(XMLdata, { compact: true, spaces: 4 }));
```

Comment:
```suggestion
        let JSONdata = JSON.parse(convert.xml2json(XMLdata, {compact: true, spaces: 4}));
```
Unnecessary await removed